packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.tsx

```diff
@@ -6,6 +6,8 @@ import {
 import { IResizeGroupProps } from './ResizeGroup.Props';
 import styles = require('./ResizeGroup.scss');
 
+const RESIZE_DELAY = 16;
```

Comment:
I picked 16ms since that's the web standard for animations. I know this isn't an animation, but I think 100ms might be noticeable to the user. Do we have similar scenarios where 100ms is acceptable?